ts-tests/tests/bridge.test.ts

```diff
@@ -82,16 +82,24 @@ describeCrossChainTransfer('Test Cross-chain Transfer', ``, (context) => {
         const provider = context.ethConfig.wallets.alice.provider;
         const currentBlock = await provider.getBlockNumber();
         await sleep(15);
+
+        let voteTransactionCount = 0;
         for (let i = currentBlock; i <= (await provider.getBlockNumber()); i++) {
             const block = await provider.getBlockWithTransactions(i);
             for (let j = 0; j < block.transactions.length; j++) {
                 if (block.transactions[j].to === context.ethConfig.bridge.address) {
                     const tx = block.transactions[j];
                     const decodedInput = inter.parseTransaction({ data: tx.data, value: tx.value });
-                    if (decodedInput.name === 'executeProposal') {
-                        const receipt = await provider.getTransactionReceipt(tx.hash);
-                        assert.equal(0, receipt.status, 'Expect the transaction fail, it actually succeeds');
-                        return;
+                    if (decodedInput.name === 'voteProposal') {
+                        voteTransactionCount++;
+                        // We have threshold = 1 in this. So it really does not matter
+                        if (voteTransactionCount === 1) {
+                            const receipt = await provider.getTransactionReceipt(tx.hash);
+                            assert.equal(1, receipt.status, 'Vote transaction should succeeds, yet the execution fails');
+                            // So the execution fails means balance no change
+                            assert.equal(handlerBalance.toString(), (await context.ethConfig.erc20.balanceOf(context.ethConfig.erc20Handler.address)).toString())
+                            return;
+                        }
```

Comment:
I don't know enough about the bridge to comment on this :sweat: 
@BillyWooo ?

Reply:
Originally, Ethereum smart contract will trigger two steps after receiving a proposal from Substrate to Relayer.
1.All relayers will make a proposal/vote for proposal until vote threshold passed.
2. One of relayers try make a seperate transaction to executed transaction.

The new smart contract (which this PR for) will:
1. relayer make/vote for a proposal, in the same transaction checking if passed the threshold and execute accordingly.

The change above is for the adjustment of new logic for test, where executeProposal is no longer existing.

Reply:
Yes, it is a rather sloppy change. Simply for the consistency of test behavior.

Reply:
executeProposal is no longer existing. It is "replaced" by the last voteProposal transaction of all relayers.

Reply:
Makes sense now; thanks for the explanation!